lib/main.dart

```diff
@@ -1,5 +1,6 @@
 import 'package:flutter/material.dart';
 import 'package:sun_flutter_capstone/consts/global_style.dart';
+import 'views/widgets/home_page.dart';
 
 void main() {
   runApp(const MyApp());
```

Comment:
suggestion to make taskbar transparent ([stackoverflow reference](https://stackoverflow.com/questions/55211640/transparent-status-bar-in-flutter#:~:text=Scaffold(%0A%20%20appBar%3A%20AppBar(%0A%20%20%20%20backwardsCompatibility%3A%20false%2C%0A%20%20%20%20systemOverlayStyle%3A%20SystemUiOverlayStyle(statusBarColor%3A%20Colors.transparent)%2C%0A%20%20)%2C%0A)))